app/models/player_profile.rb

```diff
@@ -1,3 +1,4 @@
 class PlayerProfile < ApplicationRecord
-  belongs_to :user
+  belongs_to :player
```

Comment:
we need to make sure all profiles (admin, coach, player, recruiter) that were built last night have this associated change.